agent/command/s3_put.go

```diff
@@ -410,16 +415,6 @@ retryLoop:
 
 				fpath = filepath.Join(filepath.Join(s3pc.workDir, s3pc.LocalFilesIncludeFilterPrefix), fpath)
 
-				if s3pc.skipExistingBool {
-					exists, err := s3pc.remoteFileExists(ctx, remoteName)
-					if err != nil {
-						return errors.Wrapf(err, "checking if file '%s' exists", remoteName)
-					}
-					if exists {
-						logger.Task().Infof("Not uploading file '%s' because remote file '%s' already exists. Continuing to upload other files.", fpath, remoteName)
```

Comment:
Can we keep this log in the new implementation? It's useful for users debugging their tasks.

Reply:
Yes, agreed!